cartography/intel/gcp/dns.py

```diff
@@ -191,6 +191,7 @@ def transform_dns_keys(dnsKeys: List, project_id: str, zone: Dict):
 
     for key in dnsKeys:
         key['zone'] = zone['id']
+        key['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='dns')
```

Comment:
resource_name seems inaccurate. I don't see this in cloudconsolelinks project.

Reply:
Added in console link project. Raised a PR